.github/workflows/dotnet.yml

```diff
@@ -3,20 +3,25 @@ name: .NET
 env:
   DOTNET_NOLOGO: true
   DOTNET_CLI_TELEMETRY_OPTOUT: true
-  GITHUB_USER: Nihlus
-  GITHUB_FEED: https://nuget.pkg.github.com/Remora
-  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  # GITHUB_USER: Nihlus
```

Comment:
It looks like you commented out a lot of stuff that should not have been. Fix this before I do the rest of the review, please.